### ISSUE TITLE
Batch publish/unpublish pages in Joplin

### ISSUE THREAD

**Issue by desigonz** [cityofaustin/techstack#2268]
### As a content editor, I want to be able to select multiple pages to publish or unpublish at once so that I can do my job more efficiently.

Acceptance criteria:
- A user can select multiple pages from the [content listing page](https://joplin.herokuapp.com/admin/pages/3/) and publish or unpublish them at the same time.

**Is your feature request related to a problem? Please describe.**
Right now, if we want to publish or unpublish multiple pages at once, we have to do it individually.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by desigonz** [comment]
Decided to de-prioritize and backlog. Per @toribr: "I've been doing some noodling, and am wondering if batch publishing is a good idea right now. I think your assumption is that draft pages are easily selected but in creating something that allows multiple select introduces a tool that makes huge mistakes possible, and not easily recoverable..."

**Comment by easherma** [comment]
I agree that could be handy but introduce unexpected changes. At the moment, we prompt a person each time to confirm before they unpublish. I don't think we've thought through what we'd do with that pattern in a batch context...

If an admin/editor _really_ needed to do this, they could do it using the django default admin:
https://joplin.herokuapp.com/django-admin/wagtailcore/page/

**Comment by easherma** [comment]
Syncing content, scheduling groups of content might also be some use cases
